ToDoList-frontend/src/app/bucket/bucket.component.html

```diff
@@ -276,13 +289,26 @@ <h3 class="text-3xl font-semibold">
                 <option>Cancelled</option>
               </select>      
           </div>
-            <div class="mb-6">
-          <label class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Priority</label>
-              <select id="bucketTaskPriority" name="bucketTaskPriority" formControlName="bucketTaskPriority" class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500">
-                  <option>Low</option>
-                  <option>Medium</option>
-                  <option>High</option>
-              </select>
+          <div class="mb-6">
+            <label class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Priority</label>
+            <select id="bucketTaskPriority" name="bucketTaskPriority" formControlName="bucketTaskPriority" class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500">
+                <option>Low</option>
+                <option>Medium</option>
+                <option>High</option>
```

Comment:
Ja bym te wartosci w select liscie wyswietlal na podstawie tego, co mamy w bazie. Czyli pobralbym priorytety z jakiegos endpointu z API i tutaj zrobil na nich foreach. Wtedy jak dojdzie nam nowy priorytet (np. ASAP), to wystarczy dodanie go jedynie na backendzie.

Reply:
Ok zaimplementowałem tak. Żeby dodać np. nowy priorytet czy stan wystarczy na backendzie dodać odpowiedniego enuma i dopisać do seedera go. No i jeszcze w Angularze potem ikonke kolorowa dorobić 📦